src/main/kotlin/kitchenpos/tobe/menu/infra/JpaRepositoryUtils.kt

```diff
@@ -0,0 +1,10 @@
+package kitchenpos.tobe.menu.infra
+
+import kitchenpos.tobe.menu.domain.entity.MenuV2
+import kitchenpos.tobe.menu.domain.repository.MenuRepositoryV2
+import java.util.UUID
+
+fun MenuRepositoryV2.findMenuByIdOrThrow(menuId: UUID): MenuV2 {
```

Comment:
일반적으로 find는 nullable, get은 non null 타입을 반환하는 의미로 많이 사용됩니다.

```suggestion
fun MenuRepositoryV2.getMenuById(menuId: UUID): MenuV2 {
```

non null이므로 예외를 던진다는 의미도 내포되어 있어요. 위와 같이 네이밍하는건 어떨까요?

Reply:
수정해두겠습니다~ 좋은 리뷰 감사합니다!!